index.html

```diff
@@ -246,8 +243,11 @@ <h2>Contact me</h2>
       <div id="form-container">
         <p>I'm always interested in hearing about new projects, so if you'd like touch at please get in touch.</p>
         <form action="https://formspree.io/f/mjvlqgdz" method="post">
+          <label for="name">Full Name:</label>
           <input id="name" name="Name" type="text" maxlength="30" placeholder="Full name" required/>
+          <label for="email">Email address:</label>
           <input id="email" name="email" type="email" pattern=".+@(hotmail.com|gmail.com|yahoo.com)" placeholder="Email address" required/>
+          <label for="comments">Comments:</label>
           <textarea id="comments" name="comments" cols="10" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
           <button id="btn-sub" class="button" type="submit">Get in touch</button>
         </form>
```

Comment:
- [ ] Your form input fields should have labels for the accessibility purpose :+1:

Reply:
Solved, change the visibility to show